docs/how-to-provision-using-dps.md

```diff
@@ -413,52 +437,69 @@ namespace Samples.AdtIothub
 }
 ```
 
+Save the project, then publish the function app again. For instructions on publishing the function app, see the [*Publish the app*](tutorial-end-to-end.md#publish-the-app) section of the end-to-end tutorial.
+
 ### Configure your function
 
-Next, you'll need to set environment variables in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created and the event hub. If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)) the first setting will already be configured, otherwise review both the setting up of the sample function app as well as assigning permissions to the function app.
+Next, you'll need to set environment variables in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created and the event hub. If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)) the first setting will already be configured.
 
-You will need the following values from when you set up your instance. 
-If you need to gather this value again, use the link below to the corresponding section in the setup article for finding them in the [Azure portal](https://portal.azure.com).
-* Azure Digital Twins instance **_host name_** ([find in portal](../articles/digital-twins/how-to-set-up-instance-portal.md#verify-success-and-collect-important-values))
+You will need the following values from when you set up your instance.
```

Comment:
[Duplicate of comment on similar content in line 233] I'd consider putting this in the prereqs section next to the instruction that they need an instance, and shifting the scope to "here are the values you'll need in the rest of this article from the step you just completed."
That will allow them to gather the info when they're still in the mindset of setting the instance up, before they move on to the mindset of completing the DPS connection steps, and you'll only have to write this section once instead of repeating it in both sections.

Reply:
Agree, these need to go to the prerequisites.